node/pkg/driver/node_test.go

```diff
@@ -78,6 +78,8 @@ func TestNodeStageVolume(t *testing.T) {
 	lun := 10
 	mpathDeviceName := "dm-2"
 	mpathDevice := "/dev/" + mpathDeviceName
+	rawSysDevices := "/dev/d1,/dev/d2"
```

Comment:
```suggestion
	rawSysDevices := "/dev/sd1,/dev/sd2"
```
or even sda,sdb, to stay true to the source

Reply:
done